docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  codesniffer:
+    image: php:8.2.4-cli-alpine
+    volumes:
+      - ./:/var/pdga
+    working_dir: /var/pdga
+
+  composer:
+    command: -V
+    image: composer:2.4
```

Comment:
Looks like this should work, compatibility for 2.4 is back to php 7.2 (https://github.com/composer/composer/tree/2.4.0)